src/new-components/box/box.stories.mdx

```diff
@@ -0,0 +1,41 @@
+import { Story, Meta } from '@storybook/addon-docs/blocks'
+
+<Meta title="Tips and tricks/Using the Box component" />
+
+# Tips and tricks about the `Box` component
+
+## You don't need the `classNames` helper as often now
+
+The new `Box` component's `className` prop receives the same kind of values supported by this helper. So you can do things like this:
+
+```jsx
+// This…
+<Box className={classNames(styles.container, 'custom-class-name')} />
+
+// …can be written like this
+<Box className={[styles.container, 'custom-class-name']} />
+
+// Other examples:
+<Box
+  className={[
+    styles.container,
+    isReadOnly ? styles.readOnly : null,
+    { [styles.disabled]: isDisabled },
+  ]}
+/>
+```
+
+Granted, this is only supported in the `Box`. This is because the `Box` is the only design system component that officially supports custom styles. Remember that for the other components you can use `exceptionallySetClassName`, but this one is an escape hatch, that we should aim to not use as much as the design system matures. So this one does not support the feature.
+
+## The `paddingX` and `paddingY` shorthand props
+
+The `Box` component has props to set its padding in all four direction: `paddingTop`, `paddingRight`, `paddingBottom`, `paddingLeft`. You probably have noticed that we can set them all at once with `padding`.
```

Comment:
```suggestion
The `Box` component has props to set its padding in all four direction: `paddingTop`, `paddingRight`, `paddingBottom`, `paddingLeft`. You have probably noticed that we can set them all at once with `padding`.
```

Reply:
Ricardo suggested another unrelated edit to this very line, so I'll apply your suggestion alongside Ricardo's suggestion.